ci/travis/ci.sh

```diff
@@ -237,6 +240,7 @@ build_dashboard_front_end() {
     (
       cd ray/new_dashboard/client
 
+      # shellcheck disable=SC2030
```

Comment:
Why do we need this?

Reply:
If I don't add this, shell check will failed.

Reply:
I don't think you need this because the latest master CI shows that everything is passed.

Reply:
If you don't modify the `ci.sh`, it will not be checked. `format.sh` just check the modified files.